test/runtime/Policies.go

```diff
@@ -1494,11 +1494,11 @@ var _ = Describe("RuntimePolicies", func() {
 			By("Testing hubble observe output")
 			err := hubbleRes.WaitUntilMatchFilterLine(
 				`{.source.labels} -> {.destination.ID} {.destination.labels} {.IP.destination} : {.verdict} {.event_type.type}`,
-				fmt.Sprintf("[reserved:host] -> %s [container:somelabel] %s : DROPPED 1", endpointID, endpointIP.IPV4))
+				fmt.Sprintf("[\"reserved:host\"] -> %s [\"container:somelabel\"] %s : DROPPED 1", endpointID, endpointIP.IPV4))
```

Comment:
What does this change have to do with K8s 1.19?

Reply:
Also, we could use backticks instead of double-quotes to eliminate the backslashes. Feel free to implement or ignore. I personally try to avoid backslashes :wink:

Reply:
it was due a change in the jsonpath vendor.

Reply:
+1 for @christarazi's point. If I am not wrong, maybe something like ?

```suggestion
				fmt.Sprintf(`["reserved:host"] -> %s ["container:somelabel"] %s : DROPPED 1`, endpointID, endpointIP.IPV4))
```